deepspeed/runtime/fp16/onebit/onebitadam.py

```diff
@@ -372,3 +251,31 @@ def step(self, closure=None, grads=None):
                 self.deepspeed.enable_backward_allreduce = False
 
         return loss
+
+    def load_state_dict(self, state_dict):
```

Comment:
this is the part I am not sure so Jeff/Samyam can review it closely.

Reply:
Got approval from Samyam